src/filter/fast_blur.rs

```diff
@@ -0,0 +1,163 @@
+use std::cmp::{max, min};
```

Comment:
All min/max function calls can be avoided

Reply:
fixed